community/io/src/main/java/org/neo4j/io/fs/DefaultFileSystemAbstraction.java

```diff
@@ -133,9 +134,9 @@ public void deleteRecursively( File directory ) throws IOException
     }
 
     @Override
-    public boolean renameFile( File from, File to ) throws IOException
+    public void renameFile( File from, File to, CopyOption... copyOptions ) throws IOException
     {
-        return FileUtils.renameFile( from, to );
+        FileUtils.renameFile( from, to, copyOptions );
```

Comment:
Because they take different arguments.
`FileUtils.renameFile` take a file and use `Files.move` underneath.
`Files.move` take a string which is necessary for `DelegateFileSystemAbstraction` to be able to delegate to provided `FileSystem`.